src/devices/MAX31856/README.md

```diff
@@ -0,0 +1,53 @@
+﻿# MAX31856
+
+## Summary
+The MAX31856 device is a SPI interface cold-junction compensated thermocouple to digital converter.
+
+## Sensor Image
+![Illustration of wiring from a Raspberry Pi device](device.jpg)
+
+## Usage
+The MAX31856.samples file contains a sample usage of the device. Note that this reads two temperatures. One is a connected thermocouple reading which can be read using the  ```TryGetTemperature``` command and the other is the temperature of the device itself which can be read using the ```GetColdJunctionTemperature``` command. The Cold Junction Temperature is used internally to increase the accuracy of the thermocouple but can also be read if you find a use for it.
+
+Create a new ```SpiConnectionSettings``` Class if using a Raspberry Pi do not change these settings.
+
+```csharp
+SpiConnectionSettings settings = new(0, 0)
+{
+    ClockFrequency = MAX31856.SpiClockFrequency,
+    Mode = MAX31856.SpiMode,
+    DataFlow = 0
+};
+```
+
+Create a new ```SpiDevice``` with the settings from above. Then create a new MAX31856 device with the ```SpiDevice``` as well as the correct ```ThermocoupleType``` (see note below)
+```csharp
+using SpiDevice device = SpiDevice.Create(settings);
+using MAX31856 sensor = new(device, ThermocoupleType.K);
```

Comment:
I suggest to put only general info about the sensor in this README.md and move sample code to samples/README.md (treat as optional feedback but we do that in most of the devices)